recipes/anuga/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "anuga" %}
+{% set version = "3.1.9" %}
+{% set sha = "62c45b025272bd33318b33d2fa51a9134e84254d13e8ab3814be697cdcc5074c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/anuga-{{ version }}.tar.gz
+  sha256: {{ sha }}
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [py<3.6]
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('cxx') }}      # [not win]
+    - m2w64-gcc                  # [win]
+    - libpython                  # [win]
+
+  host:
+    - python 
+    - libpython  # [win]
+    - pip
+    - cython
+    - numpy
+
+  run:
+    - python
+    - _openmp_mutex   # [linux]
```

```suggestion
  build:
    - {{ compiler('c') }}        # [not win]
    - {{ compiler('cxx') }}      # [not win]
    - m2w64-gcc                  # [win]
    - libpython                  # [win]
    - llvm-openmp  # [osx]
    - libgomp      # [linux]

  host:
    - python 
    - libpython  # [win]
    - pip
    - cython
    - numpy

  run:
    - python
```
https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp